core/src/main/java/org/axonframework/commandhandling/SimpleCommandBus.java

```diff
@@ -111,7 +116,14 @@ public SimpleCommandBus(TransactionManager transactionManager,
     @SuppressWarnings({"unchecked"})
     protected <C, R> void doDispatch(CommandMessage<C> command, CommandCallback<? super C, R> callback) {
         MessageMonitor.MonitorCallback monitorCallback = messageMonitor.onMessageIngested(command);
-        MessageHandler<? super CommandMessage<?>> handler = findCommandHandlerFor(command);
+
+        MessageHandler<? super CommandMessage<?>> handler = findCommandHandlerFor(command).orElseThrow(() -> {
+            NoHandlerForCommandException exception = new NoHandlerForCommandException(
+                    format("No handler was subscribed to command [%s]", command.getCommandName()));
```

Comment:
Shouldn't it be `No handler was subscribed **for** command`?

Reply:
Ah, I see you copied it from the `private` function below. Question still stands though.

Reply:
(Had to look this up) both are proper English, the majority of the code base uses "subscribe(d) to" in JavaDocs etc.